bin/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
You must have deleted all of the bin/ directory. You can checkout this version, then recreate bin/ and add the .gitignore back, then fast forward your HEAD and commit it back.
https://github.com/basisunus/SCIRun/blob/c24fa28687b21f646c8155fa0d8b104473d1239b/bin/.gitignore